### PR TITLE
Test

### PR DESCRIPTION
**Pull request by mindyls**
# Reviewer Checklist
https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#234

# Testing Instructions
Will fully test with 1.10.3 release:
geoips/tests/integration_tests/full_test.sh

# Summary
Please review this PR "Files changed" to ensure it includes only new changes (not changes that have already been merged through github.com PRs), and ensure that no changes that have been merged to "main' previously were removed in this PR.

NOTE: In the process of removing the "integration" branch, I branched main (which currently has all the valid 1.10.2 changes) to "test", and we will merge "test" into integration.  When resolving conflicts between test and integration, I used "--strategy ours" in order to ensure all the latest updates are retained.

Once integration is up to date, we will delete "main" and rename integration to "main".  Then we will only have a single "main" branch.

# Output
Will fully test with 1.10.3

### CODE REVIEW

File: COMMIT_MESSAGE_TEMPLATE.md [hunk 1]
```diff
@@ -28,4 +28,5 @@ Somewhere within the commit message, you must include a valid Issue ID
 * ie, like NRLMMD-GEOIPS/geoips#5
 
 NRLMMD-GEOIPS/<issue_repo_name>#<issue_num>
-
+############################################################################
```
Comment (mindyls) [inline COMMIT_MESSAGE_TEMPLATE.md:31]: This might be a mistake.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -14,5 +14,4 @@ Please see geoips/CHANGELOG_TEMPLATE.rst for instructions on updating
 CHANGELOG appropriately with each PR
 
 Release notes for previous/upcoming versions can be found in
-docs/source/releases, for reference.
-
+docs/source/releases, for reference
```
Comment (mindyls) [inline CHANGELOG.rst:17]: This might be a mistake.

**Review by evrose54 — APPROVED** [review]
These changes look good to me. Hoping this resolves our current problem of being out of sync with integration, and should make it much easier having one main branch. Good job Mindy!

File: CHANGELOG.rst (hunk 2 above)
Comment (mindyls) [reply CHANGELOG.rst:17]: Ignoring for now, no functional difference.

File: COMMIT_MESSAGE_TEMPLATE.md (hunk 1 above)
Comment (mindyls) [reply COMMIT_MESSAGE_TEMPLATE.md:31]: Ignoring for now, no functional differnece.

**Pull request merged**